src/Integration.UnitTests/Binding/SolutionBindingOperationTests.cs

```diff
@@ -214,13 +212,6 @@ public void SolutionBindingOperation_Initialization()
 
             // Assert
             testSubject.SolutionFullPath.Should().Be(Path.Combine(SolutionRoot, "xxx.sln"));
-            testSubject.Binders.Should().HaveCount(3, "Should be one per managed project");
```

Comment:
No longer relevant, as the binders are now created during Prepare phase and not Initialize